Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -5512,9 +5512,9 @@ void WebPage::reapplyEditCommand(WebUndoStepID stepID)
     if (!step)
         return;
 
-    m_isInRedo = true;
+    setIsInRedo(true);
```

Comment:
Nit: `setForScope(m_isInRedo, true, false);`?

Reply:
I personally don't really like doing that, since it's a bit added complexity / less readable imo; this is also less change to the original code too